### PR TITLE
documentation corrections/clarifications

### PR DESCRIPTION
**Pull request by rvanderwerf**
Documentation corrections and clarifications for https://github.com/micronaut-projects/micronaut-aws/issues/3

### CODE REVIEW

File: src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc [hunk 1]
```diff
@@ -46,9 +49,48 @@ aws:
...
+
+AWS Parameter Store Key: /config/application_test/simple
+AWS Parameter Store Value: some test only text
+
+
+TIP: Keep in mind certain characters are not allowed for key names: https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-parameter-name-constraints.html
+
+Check the https://github.com/micronaut-projects/micronaut-examples/aws repository for a sample application.
 
+TIP: If you are using Lambda functions with Parameter Store, you will need to manually add the 'ec2' environment to the current environments.
```
Comment (jameskleeh) [inline src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: That is pending https://github.com/micronaut-projects/micronaut-core/pull/1469

File: src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc [hunk 2]
```diff
@@ -46,9 +49,48 @@ aws:
                 # true/false - ask AWS to automatically decrypt SecureString type with KMS
                 useSecureParameters: true
                 # String - you can use a custom root path the application will look for configurations in. The default is '/config/application'.
-                rootHierarchyPath: /config/application
+                rootHierarchyPath: /config
```
Comment (jameskleeh) [inline src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: Should the default be changed?

File: src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc [hunk 3]
```diff
@@ -12,7 +12,7 @@ You can configure shared properties by going into the AWS Console -> System Mana
 
 Micronaut will use a hierarchy to read the configuration values, and supports `String`, `StringList`, and `SecureString` types.
 
-You can make environment specific configurations as well by including the environment name after an underscore `_`. For example if your `micronaut.application.name` setting is set to `helloworld` then providing configuration values under `helloworld_test` will be applied only to the `test` environment.
+You can make environment specific configurations as well by including the environment name after an underscore `_`.  Providing configuration values under `application_test` will be applied only to the `test` environment. Please note currently environment specific values of application name are not supported (i.e. `helloworld_test`).
```
Comment (jameskleeh) [inline src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: Should `helloworld_test` be supported? Is there some technical limitation?

Comment (pilot-no-21) [reply src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: #https://github.com/micronaut-projects/micronaut-aws/issues/3

I am able to access the configuration from AWS parameter store now,

I removed the explicit root hierarchy path, and named a parameter in AWS as:
config/${applicationName}/db/dbname
and accessed it using @Property(name = "db.dbname")

Comment (rvanderwerf) [reply src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: I think it should. I did testing on what was currently released and working in 1.1.0. I think maybe a previous PR might have broken that but I'm not 100% but I couldn't get it to work.

File: src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc (hunk 2 above)
Comment (rvanderwerf) [reply src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: what should it be changed to? The previous info was wrong. 'application' is put in there as default after the /config by Micronaut afaik.

File: src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc (hunk 1 above)
Comment (rvanderwerf) [reply src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: ok so should I reword this that 1.1.0 this is the case but 1.1.1+ it works? or just not mention it at all?

Comment (jameskleeh) [reply src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: This should be specified that it is only required for Micronaut 1.1.x or before

File: src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc (hunk 2 above)
Comment (jameskleeh) [reply src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc]: "you can use a custom root path the application will look for configurations in. The default is '/config/application'." So if that isn't the case then that statement should be updated

**Comment by jameskleeh** [comment]
Going to merge changes manually

**Pull request closed without merging**